app/digests/migrations/0006_auto_20180731_1352.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.7 on 2018-07-31 13:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('digests', '0005_auto_20180731_1016'),
+    ]
+
+    operations = [
```

Comment:
this and `0005` seem to change the same field, can they be consolidated into a single migration?

Reply:
Let me clean them up.